index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
+      <img src="icons/menu.png" alt="Menu Icon">
     </nav>
   </header>
-</body>
\ No newline at end of file
+  <main class="home">
+    <section class="bio">
+      <h1 class="title">I’m Jawad <br>
+        Glad to see you!</h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <h3 class="connect">LET'S CONNECT</h3>
+        <ul>
+          <li><a href="https://twitter.com/Nazari_Jawad58">
+              <img src="icons/twitter.svg" alt="twitter">
+            </a>
+          </li>
+          <li><a href="https://www.linkedin.com/in/jawad-nazari/">
+              <img src="icons/Linkedin icon.svg" alt="Linkedin">
+            </a>
+          </li>
+          <li><a href="#">
+              <img src="icons/M.png" alt="M">
+            </a>
+          </li>
+          <li><a href="https://github.com/Jawad-Nazari">
+              <img src="icons/github.svg" alt="github">
+            </a>
+          </li>
+          <li><a href="#">
+              <img src="icons/angellist.svg" alt="angellist">
+            </a>
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly try to decrease the margin between the icons here to be like the Figma Design 😃

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220177232-6b6fd477-2f06-4be0-9bb2-1861f86f3c11.png)  |  ![image](https://user-images.githubusercontent.com/83360636/220177172-1d954cc8-12d6-43e2-b3b4-6a543da47253.png)  |